src/Sulu/Bundle/ContentBundle/DependencyInjection/Configuration.php

```diff
@@ -52,6 +52,15 @@ public function getConfigTreeBuilder()
                         ->booleanNode('websocket')->defaultFalse()->end()
                     ->end()
                 ->end()
+                ->arrayNode('templates')
```

Comment:
We already have a default templates section under "structures". These define which templates should be default when new structures are created.